doc/source/whatsnew/v0.24.0.txt

```diff
@@ -704,7 +704,7 @@ Numeric
 Strings
 ^^^^^^^
 
--
+- Bug in :class:`StringMethods` where :func:`Series.str.contains` was not filling missing values with given argument for na for a categorical ``Series`` (:issue:`22158`)
```

Comment:
```suggestion
- Bug :func:`Series.str.contains` not respecting the ``na`` argument for a Categorica dtypel ``Series`` (:issue:`22158`)
```

StringMethods isn't in the API docs, so that would cause a warning.